rohmu/object_storage/google.py

```diff
@@ -299,7 +307,7 @@ def copy_file(
                 body=body,
                 destinationBucket=self.bucket_name,
                 destinationObject=destination_object,
-                sourceBucket=self.bucket_name,
+                sourceBucket=source_bucket,
```

Comment:
shouldn't this be ```source_bucket.bucket_name``` ?

Reply:
It should :(

We really need some test buckets.